sgtm/artemis/get_document_from_firestore.tpl

```diff
@@ -210,35 +209,65 @@ ___SERVER_PERMISSIONS___
       "isEditedByUser": true
     },
     "isRequired": true
-  },
-  {
-    "instance": {
-      "key": {
-        "publicId": "read_event_data",
-        "versionId": "1"
-      },
-      "param": [
-        {
-          "key": "eventDataAccess",
-          "value": {
-            "type": 1,
-            "string": "any"
-          }
-        }
-      ]
-    },
-    "clientAnnotations": {
-      "isEditedByUser": true
-    },
-    "isRequired": true
   }
 ]
 
 
 ___TESTS___
 
-scenarios: []
-setup: ''
+scenarios:
```

Comment:
This test fails: I think you've copied them from the Soteria tag and not updated them.

The mockData needs updating to have a document ID, and some of the other fields aren't needed.